data/siteText.yaml

```diff
@@ -14,20 +14,22 @@ welcome-p3: >-
 
 walleyeDecline-keynote-1: Walleye populations in many Wisconsin lakes are declining, 
 walleyeDecline-keynote-2: while largemouth bass populations in many lakes are increasing
-walleyeDecline-intro: >-
-  Walleye are a preferred species for many anglers, and important to Wisconsin’s economy and culture.
-
-walleyeDecline-p1: >-
-  Some have hypothesized that largemouth bass are directly responsible for walleye declines. 
-  However, evidence to date does not support this claim.
+walleyeDecline-l1: Anglers target both walleye and largemouth bass in Wisconsin lakes.
+walleyeDecline-l2: Walleye are preferred by many anglers, and are important to Wisconsin’s economy and culture.
+walleyeDecline-l3: Much interest exists in understanding the causes of walleye population declines and whether they can be reversed.
+walleyeDecline-l4: There are many possible explanations for the observed changes in Wisconsin lake fish communities.
+walleyeDecline-l5: Some have hypothesized that largemouth bass are directly responsible for walleye declines. However, evidence to date does not support this claim.
 
 walleyeBass-keynote-1: Walleye and largemouth bass thrive in different types of lakes
-walleyeBass-intro: >- 
-  The lake conditions that are best for one species are generally not the best for the other. 
-  Temperature is a key characteristic in determining the suitability of a lake for walleye or largemouth bass.
+walleyeBass-l1: Water temperature is key characteristic determining the suitability of a lake for different species of fish, such as walleye or largemouth bass.
 walleyeBass-interactive-setup: Can you draw the relationship between these two fish and lake temperature?
-walleyeBass-l1: Walleye prefer cool water
-walleyeBass-l2: LMB prefer warm water
+walleyeBass-l2: In general, walleye naturally reproduce more successfully in cool lakes, and largemouth bass populations are most abundant in warm lakes.
+walleyeBass-l3: >- 
+  Cool lakes that are best for walleye are the worst places for largemouth bass. 
+  At the same time, the warm lakes that are best for finding high numbers of largemouth bass have the worst conditions for walleye.
```

Comment:
I'm deferring to the fisheries biologist for this text, so we can iterate with her on the specific words used.